proof-general-pkg.el

```diff
@@ -0,0 +1,2 @@
+(define-package "proof-general" "4.4.1~pre" "A generic front-end for proof assistants (interactive theorem provers)"
+  '((emacs "24.4")))
```

Comment:
> Do you know many users of 24.3?

I don't have any stats, but is the default version in Ubuntu 14.04 LTS which is supported until 2019-04.